frontend/src/metabase/lib/auth.js

```diff
@@ -11,10 +11,9 @@ export function clearGoogleAuthCredentials() {
   }
 
   try {
-    googleAuth.signOut().then(function() {
-      console.log("Cleared Google Auth credentials.");
-    });
+    await googleAuth.signOut();
   } catch (error) {
     console.error("Problem clearing Google Auth credentials", error);
   }
+  console.log("Cleared Google Auth credentials.");
```

Comment:
I'd put this right below the `await googleAuth.signOut();` so it only gets run if that doesn't throw an error

Reply:
I copied that log without thinking much about it. Any reason I shouldn't just delete it?